js_modules/dagster-ui/packages/ui-core/src/app/InjectedComponentContext.tsx

```diff
@@ -16,18 +17,22 @@ type AComponentWithProps<Props = Record<string, never>> =
   | React.MemoExoticComponent<(props: Props) => React.ReactNode>;
 
 type InjectedComponentContextType = {
-  AppTopNavRightOfLogo?: AComponentFromComponent<typeof AppTopNavRightOfLogo> | null;
+  AppTopNavRightOfLogo: AComponentFromComponent<typeof AppTopNavRightOfLogo> | null;
   OverviewPageAlerts?: AComponentWithProps | null;
-  UserPreferences?: AComponentFromComponent<typeof UserPreferences> | null;
-  AssetsOverview?: AComponentFromComponent<typeof AssetsOverviewRoot> | null;
-  FallthroughRoot?: AComponentFromComponent<typeof FallthroughRoot> | null;
+  UserPreferences: AComponentFromComponent<typeof UserPreferences> | null;
+  AssetsOverview: AComponentFromComponent<typeof AssetsOverviewRoot> | null;
+  FallthroughRoot: AComponentFromComponent<typeof FallthroughRoot> | null;
+  AssetGraphHeader: AComponentFromComponent<typeof AssetGraphHeader> | null;
+
   RunMetricsDialog?: AComponentWithProps<{
     runId: string;
     isOpen: boolean;
     onClose: () => void;
   }> | null;
 };
-export const InjectedComponentContext = React.createContext<InjectedComponentContextType>({});
+export const InjectedComponentContext = React.createContext<InjectedComponentContextType>(
+  {} as any,
```

Comment:
using as any because I don't want to pull in the actual dependencies here. Instead we provide no default values and rely on oss/cloud providing InjectedContext explicitly.